dom.bs

```diff
@@ -6189,6 +6189,40 @@ in an <a for="/">element</a> <var>element</var>, run these steps:
 
 <hr>
 
+
+To <dfn export id=concept-element-attributes-set-by-name>create an attribute by value</dfn>
```

Comment:
Maybe we should name this "create and append an attribute"? Or maybe I don't understand "by value".

The other thing I'm thinking is that it would be nice if https://dom.spec.whatwg.org/#concept-element-attributes-set-value could invoke this algorithm as well. That would require additional arguments for namespace/prefix, but seems doable.